TYPO3.Media/Classes/TYPO3/Media/Domain/Model/Thumbnail.php

```diff
@@ -66,29 +68,38 @@ class Thumbnail implements ImageInterface
     protected $configurationHash;
 
     /**
+     * @var boolean
+     * @Flow\Transient
+     */
+    protected $async;
+
+    /**
      * Constructs a new Thumbnail
      *
      * @param AssetInterface $originalAsset The original asset this variant is derived from
      * @param ThumbnailConfiguration $configuration
      * @throws \TYPO3\Media\Exception
      */
-    public function __construct(AssetInterface $originalAsset, ThumbnailConfiguration $configuration)
+    public function __construct(AssetInterface $originalAsset, ThumbnailConfiguration $configuration, $async = false)
```

Comment:
Why not adding the $async to the ThumbnailConfiguration, I create this object to avoid having tons of properties here ... and look like I did the same things for the $transient property in the Thumbnail Generator who cause conflict with this one. In a perfect world we don't need more properties here, everything should of in the ThumbnailConfiguration